inference-engine/include/cpp/ie_infer_request.hpp

```diff
@@ -33,12 +29,16 @@ class IInferRequestInternal;
  * It can throw exceptions safely for the application, where it is properly handled.
  */
 class INFERENCE_ENGINE_API_CLASS(InferRequest) {
-    std::shared_ptr<IInferRequestInternal>          _impl;
-    std::shared_ptr<details::SharedObjectLoader>    _so;
-
-    InferRequest(const std::shared_ptr<IInferRequestInternal>&         impl,
-                 const std::shared_ptr<details::SharedObjectLoader>&   so);
+    std::shared_ptr<IInferRequestInternal>   _impl;
+    details::SharedObjectLoader              _so;
```

Comment:
`_so` should go before `_impl` or it's destructor will be called before `_impl` destructor

Reply:
fixed